.github/workflows/Release.yml

```
@@ -0,0 +1,79 @@
+          pnpm run build:plugin
+          pnpm run build:alpha > build-alpha.log 2>&1
+
+      - name: Upload build logs
+        uses: actions/upload-artifact@v4
+        with:
+          name: build-alpha-log
+          path: build-alpha.log
+
+      - name: Parse Publish tag
+        id: parse_tag
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:28: Double quote to prevent globbing and word splitting

(shellcheck)